app/views/layouts/application.html.erb

```diff
@@ -38,8 +52,9 @@
       </button>
       <ul class="dropdown-menu text-left" aria-labelledby="dropdownMenuButton1">
         <li><p class="dropdown-item"><%= current_user.email %></p></li>
-  <li><p class="dropdown-item text-decoration-none" ><%= link_to "Create Listing", new_equipment_listing_path, class: "btn text-decoration-none"%></p></li>        <li><p class="dropdown-item text-decoration-none" ><%= link_to "Pending Bookings", bookings_path, class: "btn text-decoration-none"%></p></li>
```

Comment:
Since all menu buttons have text-decoration none, it is probably better to just add a btn-top-menu class  that has no font-weight?